src/types/index.ts

```diff
@@ -0,0 +1,4 @@
+export enum ConnectFormFields {
```

Comment:
prevent using `enum` to create objects instead see https://dev.to/ivanzm123/dont-use-enums-in-typescript-they-are-very-dangerous-57bh